### PR TITLE
Build Nikau testnet genesis

### PR DESCRIPTION
**Pull request by aliXsed**
Add the Nikau test net chain spec and the generated JSON files. Here are the main settings:
**protocol id:** cennznet-nikau-v1,
**GA:** CENNZ: 16000 CPAY: 16001, 
**permissions:** root should have mint permissions,
**boot node:** /dns4/bootnode-test-0.cennznet.cloud/tcp/30333/p2p/QmNRJKZGS7ySkMSaLoJpNywgxB8vbpUsCSuFmdKpNjup4y

Note to @reviewers specially @holygits: The only change I am not sure about is the change of initial_balance to 2*INITIAL_BOND from 10u128.pow(18 + 9), that I made inspired from the Azalea config.

### CODE REVIEW

File: cli/src/chain_spec/mod.rs [hunk 1]
```diff
@@ -189,25 +190,22 @@ pub fn config_genesis(network_keys: NetworkKeys, enable_println: bool) -> Genesi
...
-				NEXT_ASSET_ID,
+			assets: vec![CENNZ_ASSET_ID, CENTRAPAY_ASSET_ID],
+			// Grant root key full permissions (mint,burn,update) on the following assets
+			permissions: vec![
+				(CENNZ_ASSET_ID, root_key.clone()),
+				(CENTRAPAY_ASSET_ID, root_key.clone()),
 			],
-			initial_balance: 10u128.pow(18 + 9), // 1 billion token with 18 decimals
-			endowed_accounts: endowed_accounts.clone(),
+			initial_balance: INITIAL_BOND * 2,
```
Comment (jordy25519) [inline cli/src/chain_spec/mod.rs]: Can set this much higher, previous value will do:  `10u128.pow(18 + 9), // 1 billion token with 18 decimals`

**Review by jordy25519 — APPROVED** [review]
Looks good to me besides the initial balance amount can be much larger on Nikau.
Would also prefer to merge #200 first

**Comment by aliXsed** [comment]
> Looks good to me besides the initial balance amount can be much larger on Nikau.
> Would also prefer to merge #200 first

@holygits now that you have replaced rimu with azalea, should I do the same and remove kauri?

File: cli/src/chain_spec/mod.rs (hunk 1 above)
Comment (aliXsed) [reply cli/src/chain_spec/mod.rs]: done

**Review by cowboy-bebug — COMMENTED** [review]
LGTM! Once https://github.com/cennznet/cennznet/pull/200 is merged, rebasing on develop wil fix the ci error

**Comment by jordy25519** [comment]
> > Looks good to me besides the initial balance amount can be much larger on Nikau.
> > Would also prefer to merge #200 first
> 
> @holygits now that you have replaced rimu with azalea, should I do the same and remove kauri?

Yeah sounds good to me!

**Review by cowboy-bebug — APPROVED** [review]
LGTM

**Pull request merged**